pkgs/servers/mail/mailpit/default.nix

```diff
@@ -1,24 +1,27 @@
-{ lib
-, stdenv
-, buildGoModule
-, nodejs
-, python3
-, libtool
-, npmHooks
-, fetchFromGitHub
-, fetchNpmDeps
-, testers
-, mailpit
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  nodejs,
+  python3,
+  libtool,
+  npmHooks,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  testers,
+  mailpit,
 }:
 
 let
-  version = "1.15.1";
+  versions = import ./versions.nix;
+
+  version = versions.version;
 
   src = fetchFromGitHub {
     owner = "axllent";
     repo = "mailpit";
     rev = "v${version}";
-    hash = "sha256-5QEn4sEZgtoFrVonZsAtvzhEkxYGDEWwhJOxqwWQJTk=";
+    hash = versions.hash;
```

Comment:
```suggestion
    inherit (source) hash;
```

Reply:
I don't really get this suggestion. Is there any performance benefit like lazy loading or is it convention or what's the reasoning?

I would rather keep using `hash = source.hash;`